### ISSUE TITLE
[Improving] Wordiness - Choppiness

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-planning-and-implementation#80]
[medium-low]

After reading the first two sections (not the introduction), I thought, "This is very choppy. It feels like it needs some words introducing the lists, like, 'Here are some resources that can help'..."
And then I thought, "This doc is supposed to be quick, short, succinct guidance. And some people think many WAI resources are too wordy. Maybe we should try using as few words as feasible in this doc -- see how it works - certainly this is the doc to try it on.

So maybe leave the first "choppy" over wordy - and do that throughout?

Some of the sections are much wordier than others. I propose taking an edit pass at significant tersification -- possibly even cutting out some content...

**Comment by shawna-slh** [comment]
For example:
- A phrase that I think can be cut: "Depending on your situation, "
- "Require less amount of time, cost, or skills to repair"

-> "Requires less time, cost, or skills to repair"
- "Once issues are prioritized, you will be better able to plan how long it will take to fix them based on the estimated effort. This gives a clear indication of what is achievable given the project resources available." Is any of this important info?
- "This guidance covered the most basic essential to help you get starting with web accessibility improvements. A broader and more comprehensive plan is required to integrate accessibility throughout the design and development process. The companion guide helps you develop such a plan:" can be made more direct and succinct
- https://github.com/w3c/wai-planning-and-implementation/issues/87

**Comment by nitedog** [comment]
Wording has been revised. Please re-open issue if changes are not adequate. Ping @slhenry
